src/test/datascience/interactiveDebugging.vscode.common.ts

```diff
@@ -58,6 +58,17 @@ export function sharedIWDebuggerTests(
                 if (options?.suiteSetup) {
                     await options?.suiteSetup(debuggerType);
                 }
+
```

Comment:
I think Don mentioned that this could be a general function. But ok to put it just here to unblock stuff.